app/controllers/salesforce_controller.rb

```diff
@@ -24,7 +24,7 @@ def create_media
     url = CreateSalesforceMediaRecord.new(article: @article, course: @course, user: @user,
                                           before_rev_id: params[:before_rev_id],
                                           after_rev_id: params[:after_rev_id]).url
-    redirect_to url
+    redirect_to url, allow_other_host: true
```

Comment:
Hmm... did you find these yourself, did tests fail to lead you to these cases of external redirects? I wonder if there are others that aren't covered.

Reply:
The tests were failing locally before I made the change, but running them now passes without this change. I must have run the test in the old branch and assumed it applies to this new branch.

Reply:
Cool. I don't see any mention of the default being changed for allow_other_hosts — https://github.com/rails/rails/releases/tag/v6.0.0.beta1 . If this is necessary, it shouldn't be changed.

Reply:
It actually exists, in the Action Pack section.

> Ensure external redirects are explicitly allowed
      Add fallback_location and allow_other_host options to redirect_to.
      Gannon McGibbon

What I am not sure is if we need to `add fallback_location`

Reply:
Ah, I guess "Ensure external redirects are explicitly allowed" means that the default changed from true to false? In Rails 5, I think this only applied to 'redirect_back' *and* it defaulted to true anyway.